### DISCUSSION TITLE
[Feature Request] Dark Theme

### DISCUSSION THREAD

**Discussion by Dvalin21** [tobychui/zoraxy#116] [Ideas]
Could you add a dark theme?  For those whose eyes are sensitive to the brightness of the page?  Thanks

**Comment by tobychui** [comment]
@Dvalin21 Well I don't have the time to add dark theme (as you can see, functional feature request are starting to pile up in the issues section). But as always, PR are welcomed :)

**Reply by Dvalin21** [reply]
No worries, these things can take a lot of time. The hope is someone who have the know how could help you with this?
